Gemfile

```diff
@@ -2,62 +2,37 @@ source "https://rubygems.org"
 git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.6.1"
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 5.2.3"
-# Use sqlite3 as the database for Active Record
+gem "bootsnap", ">= 1.1.0", require: false
+gem "coffee-rails", "~> 4.2"
+gem "jbuilder", "~> 2.5"
 gem "pg"
-# Use Puma as the app server
 gem "puma", "~> 3.11"
-# Use SCSS for stylesheets
+gem "rails", "~> 5.2.3"
 gem "sass-rails", "~> 5.0"
-# Use Uglifier as compressor for JavaScript assets
-gem "uglifier", ">= 1.3.0"
-# See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'mini_racer', platforms: :ruby
-
-# Use CoffeeScript for .coffee assets and views
-gem "coffee-rails", "~> 4.2"
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem "turbolinks", "~> 5"
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem "jbuilder", "~> 2.5"
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 4.0'
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use ActiveStorage variant
-# gem 'mini_magick', '~> 4.8'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
-
-# Reduces boot times through caching; required in config/boot.rb
-gem "bootsnap", ">= 1.1.0", require: false
-gem "rubocop", require: false
+gem "uglifier", ">= 1.3.0"
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
 end
 
 group :development do
-  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
+  gem "rubocop"
+  gem "rubocop-select"
+  gem "rubocop-checkstyle_formatter"
+  gem "saddler"
+  gem "saddler-reporter-github"
+  gem "checkstyle_filter-git"
```

Comment:
INFO: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `checkstyle_filter-git` should appear before `saddler-reporter-github`.